dockerfiles/remote-plugin-kubernetes-tooling-1.0.0/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia-endpoint-runtime:next as endpoint
+FROM quay.io/buildah/upstream:v1.8.2
+
+ENV HOME=/home/theia
+COPY --from=endpoint /home/theia /home/theia
+COPY --from=endpoint /projects /projects
+COPY --from=endpoint /etc/passwd /etc/passwd
+COPY --from=endpoint /etc/group /etc/group
+COPY --from=endpoint /entrypoint.sh /entrypoint.sh
+
+ENV KUBECTL_VERSION v1.14.1
```

Comment:
you should move these instructions before the COPY instructions to have better cache of layers